peachjam/templates/peachjam/user_following_button.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+<button class="btn btn-outline-primary ms-2"
+        type="button"
+        data-bs-toggle="modal"
+        data-bs-target="#followModal">
+  {% trans 'Follow' %}
+</button>
+<div class="modal" id="followModal" tabindex="-1">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title">{% trans 'Follow' %}</h5>
+        <button type="button"
+                class="btn-close"
+                data-bs-dismiss="modal"
+                aria-label="Close"></button>
+      </div>
+      <div class="modal-body">
+        <p>{% trans 'You are not logged in.' %}</p>
+        <p>{% trans 'To follow, you need to be logged in.' %}</p>
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">{% trans 'Close' %}</button>
+        <a href="{% url 'account_login' %}?next={{ request.path }}"
```

Comment:
```suggestion
        <a href="{% url 'account_login' %}?next={{ request.get_full_path }}"
```

so we don't lose the query string, if any